BackendApartmentReservation/BackendApartmentReservation/DataContracts/DataTransferObjects/Requests/CreateApartmentRequest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using BackendApartmentReservation.Database.Entities;
+
+namespace BackendApartmentReservation.DataContracts.DataTransferObjects.Requests
+{
+    public class CreateApartmentRequest
+    {
+        [Required]
+        public string Address { get; set; }
+
+        [Required]
+        public List<DbApartmentRoom> Rooms { get; set; }
```

Comment:
Thats very bad:D FE should have absolutely no clue about db. And in this place it explicitly says its `db`Room

Db can have much more/less info than request. So thats why we need a DTO, even if its exactly the same.

And in this case it is NOT the same, since Room has database ID in it.

Reply:
Good point. Will update :)